packages/form-core/test/validate/Validator.test.js

```diff
@@ -128,22 +145,14 @@ describe('Validator', () => {
     const tag = unsafeStatic(tagString);
 
     class MyValidator extends Validator {
+      /**
+       * @param {string} modelValue
+       * @param {string} param
+       */
       execute(modelValue, param) {
         const hasError = modelValue === 'forbidden' && param === 'values';
         return hasError;
       }
-
```

Comment:
I know these lines are not needed for the test to pass.
But since we cannot document every small feature, tests can serve here as a backup wrt how things are meant to be used.
Could yoou maybe add this info to https://github.com/ing-bank/lion/blob/master/packages/form-core/src/validate/Validator.js#L97 instead?

Reply:
Sure, will do!